src/databricks/labs/ucx/assessment/queries/database_summary.sql

```diff
@@ -0,0 +1,39 @@
+-- viz type=table, name=Database Summary, columns=database,tables,views,dbfs root,delta tables,upgrade
+-- widget title=Database Summary, col=0, row=6, size_x=3, size_y=8
+SELECT `database`,
+       SUM(IS_TABLE) AS TABLES,
+       SUM(IS_VIEW) AS VIEWS,
+       SUM(IS_DBFS_Root) AS `DBFS Root`,
+       SUM(IS_DELTA) AS `Delta Tables`,
+       CASE
+           WHEN (SUM(IS_DBFS_Root)/SUM(IS_TABLE) > .3) THEN "Asset Replication Required"
+           WHEN (SUM(IS_DELTA)/SUM(IS_TABLE) < .7) THEN "Some Non Delta Assets"
+           ELSE "In Place Sync"
+       END AS Upgrade
+FROM
+  (SELECT DATABASE,
+          name,
+          object_type,
+          UPPER(table_format) AS format,
+          LOCATION,
+          CASE
+              WHEN object_type IN ("MANAGED",
+                                   "EXTERNAL") THEN 1
+              ELSE 0
+          END AS is_table,
+          CASE
+              WHEN object_type="VIEW" THEN 1
+              ELSE 0
+          END AS is_view,
+          CASE
+              WHEN STARTSWITH(location, "/dbfs/")
```

Comment:
First startswith is redundant

Reply:
I don't think so, as this captures all locations pointing to `dbfs` but are not mounts. If you were to drop this first condition, all other locations would be classified as "DBFS Root", which is wrong.